spec/requests/api/v1/moves_controller_show_spec.rb

```diff
@@ -44,20 +40,12 @@
       let(:move_id) { 'UUID-not-found' }
 
       it_behaves_like 'an endpoint that responds with error 404'
-
-      it 'doesn\'t sync data' do
-        expect(Moves::NomisSynchroniser).not_to have_received(:new)
-      end
     end
 
     context 'with an invalid CONTENT_TYPE header' do
       let(:content_type) { 'application/xml' }
 
       it_behaves_like 'an endpoint that responds with error 415'
-
-      it 'doesn\'t sync data' do
-        expect(Moves::NomisSynchroniser).not_to have_received(:new)
-      end
```

Comment:
Is this test not needed anymore?

Reply:
Well what's the point of expecting "not to" do something?

Reply:
If, for example, we wanted to assert that the NomisSynchroniser wasn't called inappropriately. What was the rationale when this test was added. It'd be good to understand that in order to be comfortable removing this.

Reply:
If I remember correctly, we used to sync with Nomis under certain conditions in the /show action, but that has been removed when we discovered it was messing with duplicates. The tests don't fail because they weren't very relevant at the time and they're even less relevant now. There's no reference to the `NomisSynchroniser` right now in the whole action.

Reply:
> There's no reference to the NomisSynchroniser right now in the whole action.

Sounds good to remove it then :+1: